PeerioTechnologies/peerio-desktop
# feat: redesigned UI for message send error actions

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/9689/desktop-message-errors

#### Testing instructions  
Failed message (e.g. disconnect from internet and try to send a message) has been redesigned. Same functions, but they're more obvious now, instead of being in a hidden dropdown menu.

Note that failed inline files (via upload or share) are not included in this PR, contrary to the mocks. This is much bigger task and likely needs a different design. Paul will make a new story.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
